### PR TITLE
Add CloudFormation template size limit

### PR DESCRIPTION
**Pull request by skuzzer**
*Issue #, if available:*

*Description of changes:* Added limit for CloudFormation template file size when using AWS CloudFormation provider

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/limits.md [hunk 1]
```diff
@@ -23,4 +23,5 @@ The following table lists current limits in AWS CodePipeline\. Structural requir
...
 |  Maximum size of the JSON object that can be stored in the `ParameterOverrides` property  | For an AWS CodePipeline deploy action with AWS CloudFormation as the provider, the ParameterOverrides property is used to store a JSON object that specifies values for the AWS CloudFormation template configuration file\. There is a maximum size limit of 1 kilobyte for the JSON object that can be stored in the ParameterOverrides property\. | 
-|  Maximum size of the image definitions JSON file used in pipelines deploying Amazon ECS containers and images  | 100 KB | 
\ No newline at end of file
+|  Maximum size of the image definitions JSON file used in pipelines deploying Amazon ECS containers and images  | 100 KB | 
+|  Maximum size of CloudFormation template file  | For an AWS CodePipeline deploy action with AWS CloudFormation as the provider there is a maximum size limit of 51,200 bytes for the template file\. |
```
Comment (darlaker) [inline doc_source/limits.md:27]: Thank you for this contribution. We are reviewing this change.

Comment (skuzzer) [reply doc_source/limits.md:27]: This should no longer apply, this message indicates this limit has been removed https://forums.aws.amazon.com/message.jspa?messageID=842437#842437

Comment (darlaker) [reply doc_source/limits.md:27]: Thank you for your reply and update on this. This update is correct. The template size limit has been removed. Thank you again for your contribution to the AWS CodePipeline User Guide.

**Comment by darlaker** [comment]
Thanks again for submitting this feedback. Since this is no longer an issue, will close this PR. Keep the feedback coming!

**Pull request closed without merging**